tidy3d/components/scene.py

```diff
@@ -984,15 +999,37 @@ def _add_cbar_eps(eps_min: float, eps_max: float, ax: Ax = None) -> None:
         )
 
     @staticmethod
-    def _eps_bounds(medium_list: list[Medium], freq: float = None) -> Tuple[float, float]:
+    def _eps_bounds(
+        medium_list: list[Medium], freq: float = None, component: str = None
+    ) -> Tuple[float, float]:
         """Compute range of (real) permittivity present in the mediums at frequency "freq"."""
         medium_list = [medium for medium in medium_list if not medium.is_pec]
-        # regular medium
-        eps_list = [
-            np.real(medium.eps_model(freq))
-            for medium in medium_list
-            if not isinstance(medium, AbstractCustomMedium) and not isinstance(medium, Medium2D)
-        ]
+        if component is None:
+            # user has not selected a permittivity component, treat all as regular media
+            eps_list = [
+                np.real(medium.eps_model(freq))
+                for medium in medium_list
+                if not isinstance(medium, AbstractCustomMedium) and not isinstance(medium, Medium2D)
+            ]
+        else:
+            # regular media
+            eps_list = [
+                np.real(medium.eps_model(freq))
+                for medium in medium_list
+                if not isinstance(
+                    medium,
+                    (AbstractCustomMedium, Medium2D, AnisotropicMedium, FullyAnisotropicMedium),
+                )
+            ]
+            # anisotropic medium with user selected component
+            comp2indx = {"x": 0, "y": 1, "z": 2}
+            eps_list += [
+                medium.eps_comp(
+                    row=comp2indx[component[0]], col=comp2indx[component[1]], frequency=freq
```

Comment:
assumes too much of `component`. Would give a hard to understand KeyError if eg. `component="fg"`. we should check right away I think to ensure that `len(component) == 2 and all(c in 'xyz' for c in component)` if `component is not None`